core/services/remote_access_services_test.py

```diff
@@ -0,0 +1,103 @@
+# coding: utf-8
+#
+# Copyright 2017 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for remote access services."""
+
+import json
+
+from core.services import remote_access_services
+from core.tests import test_utils
+import vmconf
+
+class RemoteAccessServicesTests(test_utils.GenericTestBase):
```

Comment:
We don't have continuous integration, for now, so till then, we gotta make sure that all the tests are actually passing before every push!